src/fuel_fab.cc

```diff
@@ -507,6 +507,13 @@ void FuelFab::RecordPosition() {
       ->Record();
 }
 
+void FuelFab::Tick() {
+  cyclus::toolkit::RecordTimeSeries<double>(fuelfab_record_input, this, 
+                                            throughput);
+  cyclus::toolkit::RecordTimeSeries<double>(fuelfab_record_output, this, 
```

Comment:
hmm shouldn't these be the input and output commodity names?

Reply:
D3ploy needs unique calls for these.  A fuelfab and a reactor might share the same commodity to trade fuel, but in order to compare the supply and demand of that commodity the facilities need their own unique tags for that commodity.